### DISCUSSION TITLE
Add pull only feature

### DISCUSSION THREAD

**Discussion by Jogi7819** [devcontainers/community#46] [Ideas]
Hi together,

we use the devcontainer cli on Jenkins as well in a pipeline with a high degree of parallization, in which we use for each parallel branch a different container created from the same devcontainer.json file.

As of now the pipeline has an initial prepare stage where all the containers are created in a sequence. To reduce the overall execution time of the pipeline one could pull the image from the remote to the local cache ones and create the containers parallel during parallel stage execution. My proposal is to add something like
```
devcontainer pull
``` 
or
```
devcontainer up --pull-image-only
```

BR Jochen

**Comment by bamurtaugh** [comment]
Thanks for opening this! @chrmarti do you think this would make sense as a feature request in https://github.com/devcontainers/cli/issues?
